### DISCUSSION TITLE
Adding a search bar in the library section

### DISCUSSION THREAD

**Discussion by hakkm** [excalidraw/excalidraw#8217] [General]
When i downloaded an IT library, i needed to search for components as fast as possible, because i don't want to think over about where  the icon is located. 
i think it will be also a good idea to have a shortcut like ctrl+/ but it opens a search bar for library componenents

**Comment by egor-inf** [comment]
Yes, this would be a killer feature - in drawio it's extremely easy to find elements

**Comment by mohsinmdl** [comment]
when should we expect this feature in place, love the tool though :)

**Comment by Mrazator** [comment]
We will definitely add this at some point, but we need to finish some perquisites first!

**Reply by Mrazator** [reply]
https://github.com/excalidraw/excalidraw/issues/8304

**Comment by LordMoMA** [comment]
still no follow up?
